### ISSUE TITLE
Multi Tenant Example login not as documented 

### ISSUE THREAD

**Issue by morja** [payloadcms/payload#4938]
# Documentation Issue

In the docs it says the login for the multi-tenant example is dev@payloadcms.com with pw test but when I run the example the user in mongodb is demo@payloadcms.com with pw demo.

## Additional Details

Maybe I did something wrong?
I did:
git clone git@github.com:payloadcms/payload.git
cd payload/examples/multi-tenant/
cp .env.example .env
yarn && yarn dev

**Comment by andymcbee** [comment]
I ran into this too when first trying to run the example. I was able to get by it by reading the source code. Would it be OK if I fixed this to help improve DX for future users? @kendelljoseph @jmikrut 

It's a simple documentation tweak but I'm trying to get my hands dirty with some OS contributions.  Would appreciate the opportunity to experience the flow of submitting a PR on here.

I'm a 4th year comp sci student with a few years of dev experience. I'm in the process of migrating away from Strapi on one of my prod apps and incorporating PayloadCMS instead, which is how I stumbled across the documentation error.
